Configure

```diff
@@ -812,6 +823,13 @@ if ($libs =~ /(^|\s)-Wl,-rpath,/
 	"***** any of asan, msan or ubsan\n";
 }
 
+if ($#seed_sources < 0) {
+    die "No seed config and no defaults set";
```

Comment:
Where would the defaults come from?  As it currently stands, this should make *all* our builds fail on configuration.